### PR TITLE
GAD-1: Made static tabs editable dynamically for CSE dapartment.

### PR DESCRIPTION
**Pull request by CHOUHAN-MOHIT**
### faculty now can edit about us , facility and achievement tabs.

-   [x] faculty now can edit about us , facility and achievement tabs.
-   [x] faculty can add announcement directly from announcement tab.
-   [x] now faculty information will be shown in cards.

## Screenshots
WhatsApp Image 2023-04-04 at 14 51 22
WhatsApp Image 2023-04-04 at 14 54 43
WhatsApp Image 2023-04-04 at 14 55 03
WhatsApp Image 2023-04-04 at 14 58 26
WhatsApp Image 2023-04-04 at 14 56 25

### CODE REVIEW

**Review by Sha2nkGitHub — APPROVED** [review]

**Comment by CHOUHAN-MOHIT** [comment]
@PranshuNayak  This pr also creates a new model names department which will be used to contain department information like about , facility and achievements. Have a look and let me know if anything needs to be changed.

**Comment by akshatnema** [comment]
@CHOUHAN-MOHIT Kindly resolve the merge conflicts.

**Comment by CHOUHAN-MOHIT** [comment]
@akshatnema all the conflicts have been resolved please check it now.

**Comment by akshatnema** [comment]
Changes in models.py. review required from @PranshuNayak and @samay-rgb.

@CHOUHAN-MOHIT kindly undo the changes made in development.py file.

File: FusionIIIT/applications/department/admin.py [hunk 1]
```diff
@@ -1,7 +1,8 @@
...
 
 # Register your models here.
-from .models import(Announcements, SpecialRequest)
+from .models import(Announcements, SpecialRequest,Department)
 
 admin.site.register(Announcements)
-admin.site.register(SpecialRequest)
\ No newline at end of file
+admin.site.register(SpecialRequest)
+admin.site.register(Department)
```
Comment (PranshuNayak) [inline FusionIIIT/applications/department/admin.py:8]: remove the indentation changes and only push changes related to Department

**Review by PranshuNayak — COMMENTED** [review]
Don't push indentation changes.

**Review by PranshuNayak — APPROVED** [review]

**Review by samay-rgb — APPROVED** [review]

**Comment by Sha2nkGitHub** [comment]
ping @akshatnema @manavdesai27

**Pull request merged**